team/src/components/TeamTypingIndicator/TeamTypingIndicator.css

```diff
@@ -0,0 +1,71 @@
+.typing-indicator__input {
+  position: absolute;
+  bottom: calc(100% - 880px + 7px);
```

Comment:
not sure this is how we want to do it. this works in a fixed height layout, but not in the responsive one we're going for, ideally it's just part of the messageinput, and will be positioned using some margin/padding.